### DISCUSSION TITLE
Enable/bring back "pulsing/glowing" alerting for dashboard panel

### DISCUSSION THREAD

**Discussion by nikobearrr** [grafana/grafana#40978] [Alerting]
**What happened**:

In Grafana 6.x when you have an alerting dashboard the whole dashboard panel would glow/pulse in red showing you something is wrong. That seems to have been removed from 7.x

**What you expected to happen**:

I would like to have the option of enabling the said "pulsing" as on some dashboards it's hard to spot if there/where is the red heart  

**How to reproduce it (as minimally and precisely as possible)**:

On a 4K monitor it's sometimes really hard to spot the alerting dashboard as the heart is very small

![Screenshot 2020-05-31 at 21 17 26](https://user-images.githubusercontent.com/8693064/83358524-3269e500-a385-11ea-9d5d-a684a1cd5ae1.png)

**Anything else we need to know?**:

**Environment**:
- Grafana version: 7.0.2

**Comment by dprokop** [comment]
@nikobearrr thanks for reporting this. I can confirm that this is a regression.

**Comment by nikobearrr** [comment]
Hi @torkelo, I can see you have put the "awaiting-more-evidence". If you think my description is not enough or not understandable enough I can help out and try to record some examples

**Comment by torkelo** [comment]
I mean awaiting more demand / need for this

**Comment by ivanahuckova** [comment]
@nikobearrr I know this is not 100% what you have in mind, but I hope that this [PR](https://github.com/grafana/grafana/pull/25599) is going to improve your experience at least a little. 🙂 We have improved alerting icons, and you should be able to see alerts easier.

**Comment by nikobearrr** [comment]
Hi @ivanahuckova , thanks for the heads up. I think that would help a bit. It still doesn't completely solve the issue as an icon is hard to see on a tv/pc high resolution screen.

Our use case is that we let a dashboard run with time - 6h. An alert might have occurred outside of time range, so no red dashed line can be seen. The only way to find out its alerting is by seeing this 10x10 (or perhaps more) icon on a 4K TV. This is where panel pulsing/border color changing helps

**Comment by cartw** [comment]
I would also like to see the pulsing border return. When viewing dashboards on a personal screen there's not much of an issue, but for more distant screens a small icon is really not enough to draw attention to a particular panel.

**Comment by SuperAvarice** [comment]
+1 for this.
For large screens this change makes the distinction very hard to see.
Perhaps an ini setting for enabling this?

**Comment by b-boyd** [comment]
+1 for wanting the pulsing border to return. It visually stood out more than the small heart, more quickly catching one's attention of where on the dashboard to focus.

**Comment by orojina** [comment]
+1 to bring back the glowing alerting panels.

**Comment by armandgrillet** [comment]
Due to the "+1"s and the possible implementations now that we have the new alerting system, I am moving this to a GH discussion.

**Comment by cartw** [comment]
So the issue was closed and this discussion created, but not a lot of discussion happening. I, for one, would still like the way an alerting panel is highlighted to be a lot more obvious than just the small heart icon. 
I don't know if there are other accessibility issues (e.g. colour blindness) around such a tiny visual element switching from green to red as the sole indicator?

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!

cc @grafana/alerting-squad
